airbyte-integrations/connectors/source-hubspot/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "4.2.3"
+version = "4.2.3dev1"
```

Comment:
Please bump the version correctly to `4.3.0`, also bump the metadata file and doc entry changelog.

Reply:
After run `poetry lock` in the connector folder to regenerate the dependencies.